polkadot/node/network/bridge/src/rx/mod.rs

```diff
@@ -1038,21 +1041,58 @@ fn dispatch_collation_event_to_all_unbounded(
 	}
 }
 
+fn try_send_validation_event<E>(
```

Comment:
I find the name a bit misleading. The `try` suggest that we don't wait when full - which is true, but we also don't give up. Maybe send_or_queue_validation_event?